build.gradle

```diff
@@ -455,6 +455,12 @@ sourceSets {
             runtimeClasspath += sourceSets.patchedMc.output + sourceSets.mcLauncher.output
         }
     }
+    // This ensures that VSCode knows where the minecraft classes are
+    main {
+        java {
+            compileClasspath += sourceSets.patchedMc.output + sourceSets.mcLauncher.output
```

Comment:
This is already done by RFG here https://github.com/GTNewHorizons/RetroFuturaGradle/blob/master/plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/mcp/MCPTasks.java#L938, I'm not sure why VSC doesn't pick up on it?

Reply:
From what I can tell. VSC sees the project as a gradle project and searches sourceSets for any outside sources. It only searches the build.gradle for sources not given through dependencies.gradle.

Reply:
before my change on a redownloaded gt5u
![image](https://user-images.githubusercontent.com/76872108/227044453-def2f36d-1539-4580-980f-38b7b1ec733f.png)

with my change
![image](https://user-images.githubusercontent.com/76872108/227041739-ff3cc6f2-5ec1-43bf-aabb-e73ffd952e7f.png)

Reply:
*sometimes its hard to debug things with VSC as it decided that it will sometimes just work*

Reply:
the sources look correct to me on the first picture, what even is `bin/generated-sources` as gradle does not use `bin` - do you have an override for the build folder set somewhere?

Reply:
Anyway I will try moving the source set classpath initalization earlier in RFG and let's see if that works, because that's where it should be done, not in the buildscript.